pinc/stages.inc

```diff
@@ -356,8 +356,10 @@ function can_user_get_pages_in_project($username, $project, $round)
         }
     }
 
-    // In P1, English and French projects, for a while after their release,
-    // are reserved for inexperienced users.
+    // In P1, English projects, for a while after their release,
```

Comment:
I would make this comment generic and not include any of the possible reasons and defer that entirely to `get_reserve_length()` so we don't need to keep them in sync.

Reply:
Good idea, and done.